boskos/cleaner/cmd/main.go

```diff
@@ -41,11 +41,15 @@ var (
 	kubeClientOptions crds.KubernetesClientOptions
 
 	boskosURL    string
+	username     string
+	password     string
 	cleanerCount int
 )
 
 func init() {
 	flag.StringVar(&boskosURL, "boskos-url", "http://boskos", "Boskos Server URL")
+	flag.StringVar(&username, "username", "", "Username used to access the Boskos server")
+	flag.StringVar(&password, "password", "", "Password used to access the Boskos server")
```

Comment:
Generally we do not ask for users to provide sensitive data in a flag. Instead, we ask for a path to the file containing the data. Using the secret agent allows us to load this, re-load it if it changes and also plug into the `logrus` implementation to censor secret data.

Reply:
Understood.